test/mode/modeNormal.test.ts

```diff
@@ -277,327 +290,218 @@ suite("Mode Normal", () => {
-
-        await assertEqualLines([" text"]);
-    });
-
-
-    test("Can handle 'dt'", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'iaext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '^',
-            'd', 't', 't'
-        ]);
-
-        await assertEqualLines(["t text"]);
-    });
-
-    test("Can handle 'f' twice", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '^',
-            'f', 't',
-            'f', 't'
-        ]);
-
-        assertEqual(TextEditor.getSelection().start.character, 5, "f failed");
-    });
-
-    test("Can handle 'F'", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '$',
-            'F', 't'
-        ]);
-
-        assertEqual(TextEditor.getSelection().start.character, 5, "F failed");
-    });
-
-    test("Can handle 'F' twice", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '$',
-            'F', 't',
-            'F', 't',
-        ]);
-
-        assertEqual(TextEditor.getSelection().start.character, 3, "F failed");
-    });
-
-
-
-
-    test("Can handle 't'", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '^',
-            't', 't'
-        ]);
-
-        assertEqual(TextEditor.getSelection().start.character, 2, "f failed");
-    });
-
-    test("Can handle 't' twice", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '^',
-            't', 't',
-            't', 't'
-        ]);
-
-        // it does nothing the second time lawl
-        assertEqual(TextEditor.getSelection().start.character, 2, "f failed");
-    });
-
-    test("Can handle 'T'", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
-
-        await modeHandler.handleMultipleKeyEvents([
-            '<esc>',
-            '$',
-            'T', 't'
-        ]);
-
-        assertEqual(TextEditor.getSelection().start.character, 6, "F failed");
-    });
-
-    test("Can handle 'T' twice", async () => {
-        await modeHandler.handleMultipleKeyEvents(
-            'itext text'.split('')
-        );
+    newTest("$ always keeps cursor on EOL",
+        {
+            start: ['text text', 'text', 'text tex|t'],
```

Comment:
You can use backtick strings in JavaScript/TypeScrip. They can span multiple lines.